examples/agents/in-house-ctf/README.md

```diff
@@ -0,0 +1,57 @@
+# In House CTF Tasks
+This directory includes Capture The Flag challenges migrated from Google DeepMind's [Dangerous Capabilities Evaluations](https://github.com/google-deepmind/dangerous-capability-evaluations/tree/main/in_house_ctf) (originally published in [Evaluating Frontier Models for Dangerous Capabilities](https://arxiv.org/pdf/2403.13793)). Here we include 9 of the 13 In-House CTF challenges (excluding 4 challenges that have faulty Docker images from the original repository). 
```

Comment:
Is it still the case that we are excluding 4 images?

Reply:
Ah good catch, forgot to update this bit